packages/expo-cli/lib/version-information.js

```diff
@@ -1,7 +1,7 @@
 const semver = require('semver')
 
 // the major version number of the latest Expo SDK we support
-const LATEST_SUPPORTED_EXPO_SDK = 47
+const LATEST_SUPPORTED_EXPO_SDK = 48
```

Comment:
@tomlongridge does 45 become legacy at this stage? I seem to recall we'd agreed on a one in, one out policy

Reply:
Yes - we're matching this: https://docs.expo.dev/workflow/upgrading-expo-sdk-walkthrough/

Might be worth updating the contributing guide to say what's needed to make it so.

From the wording on Expo's side, it's only ~6 months, it won't necessarily be one-out-one-in.

Reply:
I'm not sure I understand what else needs to change here - what does legacy mean for us? Does that mean we will no longer support Expo 45? 

We do have a v45 version that would get installed, so feels like adding it to the `legacyVersions` list here would be purely for documentation purposes?

Reply:
I don't think so? unless we should document somewhere about which branches we need to manage from here? will 45 and 45-next continue to be updated after this point?

Reply:
> I don't think so? unless we should document somewhere about which branches we need to manage from here? will 45 and 45-next continue to be updated after this point?

We don't have to do anything but we could delete the branches we're no longer maintaining since we can recreate them from the tags if we need to (e.g. for a security issue or other critical fix)